pkg/acceptance/keycloak/realm-export.json

```diff
@@ -0,0 +1,2450 @@
+{
```

Comment:
this is a really long json. I have a few questions:
- Do we need all of it?
- Isn't there any sensitive data leaked here?